public/js/question.js

```diff
@@ -21,3 +21,28 @@ $('#answerform').submit(function(evt) {
         }
     });
 });
+
+
+
+$('#mc_answerform').submit(function(evt) {
+    evt.preventDefault();
+    var ans = $("input[name=answer]:checked").val();
+    $.ajax({
+        type: 'POST',
+        url: '/submitREanswer',
+        data: $('#mc_answerform').serialize(),
+        success: function(data) {
+            if (data.result == 'incorrect') {
```

Comment:
This looks like it's duplicated from submitting a short answer. can we combine the 2 and add a flad for the type?

Reply:
I have thought about that as  well, but the problem we run in to is scalability of adding more question types, and for that each question has a different way of collecting the answer. In this case for MC and RE questions the method of checking the answer is the same.

Reply:
I agree with mehtadarshan1. Backend will be doing the same thing as well. Different path for different type of question